gcc/rust/lex/rust-lex.h

```diff
@@ -16,7 +16,14 @@ struct RAIIFile
   FILE *file;
 
 public:
-  RAIIFile (const char *filename) : file (fopen (filename, "r")) {}
+  RAIIFile (const char *filename)
+  {
+    if (strncmp (filename, "-", 1) == 0)
```

Comment:
It doesn't matter in practice (`error: unrecognized command-line option '-.rs'`, etc.), but why `strncmp` with a length of one, which matches every `filename` beginning with `-`?  Shouldn't this instead use `strcmp (filename, "-")` to exactly match `-`?

Reply:
good point I missed that in the review.

Reply:
Ah, fair. I didn't think about files starting with `-`. I'll fix it right away.